airbyte-json-validation/src/main/java/io/airbyte/validation/json/JsonSchemaValidator.java

```diff
@@ -68,7 +68,7 @@ public boolean test(final JsonNode schemaJson, final JsonNode objectJson) {
     final Set<ValidationMessage> validationMessages = validateInternal(schemaJson, objectJson);
 
     if (!validationMessages.isEmpty()) {
-      LOGGER.info("JSON schema validation failed. \nerrors: {}", Strings.join(validationMessages, ", "));
+      LOGGER.info("JSON schema has some validation warnings. \nFields need to be updated in your configuration: {}", Strings.join(validationMessages, ", "));
```

Comment:
I'm not sure whether we should add 'fields need to be updated in your configuration' -- I think sometimes this is an issue with connector configuration, but it could also be an issue from the api/db source itself. 

Maybe something like  "JSON schema has some validation warnings. Record schema does not conform to configured connector schema." just to give them the info that it's related to a record schema.